linters.go

```diff
@@ -53,27 +53,29 @@ var predefinedPatterns = map[string]string{
 	"PATH:LINE:MESSAGE":     `^(?P<path>.*?\.go):(?P<line>\d+):\s*(?P<message>.*)$`,
 }
 
-func getLinterByName(name string, customSpec string) *Linter {
-	if customSpec != "" {
-		return parseLinterSpec(name, customSpec)
+func getLinterByName(name string, overrideConf LinterConfig) *Linter {
```

Comment:
Test coverage looks good, I think the only case we're missing is a test that shows that this can set the Pattern.

I don't think we need another one that goes through `setupFlags`, so a test like `TestGetLinterByName()` that passes in a config and shows that a default pattern is overridden would be great.